notes/returned-mail.vbs

```diff
@@ -214,20 +213,43 @@ IF ADDR_actions <> "no response received" THEN
     		IF mail_line_one <> "" THEN
     			IF mailing_address_confirmed = "Select:" THEN err_msg = err_msg & vbCr & "Please confirm if the mailing address is the address that the agency attempted to deliver mail to."
     		END IF
-    		IF mailing_address_confirmed = "NO" and residential_address_confirmed = "NO" and notes_on_address = "" THEN  err_msg = err_msg & vbCr & "Please confirm what the address was using notes on address that the agency attempted to deliver mail to. ADDR will not be updated at this time. Please explain where the address was found and on what date."
-    		IF returned_mail = "" THEN  err_msg = err_msg & vbCr & "Please explain what mail was returned."
-    		IF isdate(due_date) = FALSE THEN err_msg = err_msg & vbnewline & "Please enter the verifications requested due date."
+			IF residential_address_confirmed = "YES" and mailing_address_confirmed = "YES" THEN err_msg = err_msg & vbCr & "Please confirm what the address the agency attempted to deliver mail to."
```

Comment:
Line 213 - Test case #305410

- Allowed me to select the mailing addr even though it was just a comma
- comma on mailing addr - can a trim and replace be added to clear out the mailing addr?
- Returned mail allows me to select no on resi addr and nothing on mailing addr, and will case note just the comma.

Reply:
trim(mail_city_line) added 
IF residential_address_confirmed = "NO" and mailing_address_confirmed <> "YES" added